tests/test-daemon-integration.py

```diff
@@ -347,7 +346,7 @@ def test_record_singular_event_has_reasonable_relative_timestamp(self):
 
         relative_time_difference = relative_time_second - relative_time_first
         self.assertLessEqual(0, relative_time_difference)
-        absolute_time_difference = self._NANOSECONDS_PER_SECOND * \
+        absolute_time_difference = round(1e9) * \
```

Comment:
Oops, I was under the mistaken impression that `time.time()` returned an `int`. Yeah, this is kinda silly.